tests/ops/fixtures/application_fixtures.py

```diff
@@ -592,6 +590,291 @@ def erasure_policy_two_rules(
+            "policy_id": erasure_policy.id,
+            "masking_strategy": {
+                "strategy": StringRewriteMaskingStrategy.name,
+                "configuration": {"rewrite_value": "Masked"},
+            },
+        },
+    )
+    user__contact__address__state_target = RuleTarget.create(
+        db=db,
+        data={
+            "client_id": oauth_client.id,
+            "data_category": DataCategory("user.contact.address.state").value,
+            "rule_id": user__contact__address__state_rule.id,
+        },
+    )
+    user__contact__address__street_rule = Rule.create(
+        db=db,
+        data={
+            "action_type": ActionType.erasure.value,
+            "client_id": oauth_client.id,
+            "name": "user__contact__address__street Erasure Rule",
+            "policy_id": erasure_policy.id,
+            "masking_strategy": {
+                "strategy": StringRewriteMaskingStrategy.name,
+                "configuration": {
+                    "rewrite_value": "Masked Masked Masked Masked Masked Masked"
+                },
+            },
+        },
+    )
+    user__contact__address__street_target = RuleTarget.create(
+        db=db,
+        data={
+            "client_id": oauth_client.id,
+            "data_category": DataCategory("user.contact.address.street").value,
+            "rule_id": user__contact__address__street_rule.id,
+        },
+    )
+    user__date_of_birth_rule = Rule.create(
+        db=db,
+        data={
+            "action_type": ActionType.erasure.value,
+            "client_id": oauth_client.id,
+            "name": "user__date_of_birth Erasure Rule",
+            "policy_id": erasure_policy.id,
+            "masking_strategy": {
+                "strategy": StringRewriteMaskingStrategy.name,
+                "configuration": {"rewrite_value": "1/1/1900"},
+            },
+        },
+    )
+    user__date_of_birth_target = RuleTarget.create(
+        db=db,
+        data={
+            "client_id": oauth_client.id,
+            "data_category": DataCategory("user.date_of_birth").value,
+            "rule_id": user__date_of_birth_rule.id,
+        },
+    )
+    user__name_rule = Rule.create(
+        db=db,
+        data={
+            "action_type": ActionType.erasure.value,
+            "client_id": oauth_client.id,
+            "name": "user__name Erasure Rule",
+            "policy_id": erasure_policy.id,
+            "masking_strategy": {
+                "strategy": StringRewriteMaskingStrategy.name,
+                "configuration": {"rewrite_value": "Masked"},
+            },
+        },
+    )
+    user__name_target = RuleTarget.create(
+        db=db,
+        data={
+            "client_id": oauth_client.id,
+            "data_category": DataCategory("user.name").value,
+            "rule_id": user__name_rule.id,
+        },
+    )
+
+    yield erasure_policy
+
+    try:
+        user__device__ip_address_rule.delete(db)
```

Comment:
We need to wrap these in individual try/except blocks because if there's an exception thrown at the beginning of the list, then the rest of the deletes won't execute